tests/fsharpqa/Source/InteractiveSession/Misc/GenericConstraintWoes/issue2411/app.fsx

```diff
@@ -0,0 +1,10 @@
+#r "lib.dll"
+
+open System.Reflection
+
+type FooImpl() =
```

Comment:
Foo triggers a failure on policheck which is one of our tools that validate code for offensive words,  please, let's not introduce any new occurrences of it.

Thanks

Kevin